packages/components/tokens/components/checkbox.js

```diff
@@ -6,31 +6,31 @@ const checkmarkSvg =
 module.exports = {
   base: {
     border: { value: `1px solid` },
-    borderColor: { value: `${elements.color.gray}` },
+    borderColor: { value: `${elements.color.gray.base}` },
     background: { value: `#fff` },
     easing: { value: `${elements.motion.easing()}` },
     focus: {
-      outline: { value: `1px dotted ${elements.color.primary}` },
+      outline: { value: `1px dotted ${elements.color.primary.base}` },
     },
     hover: {
-      borderColor: { value: `${elements.color.gray}` },
-      background: { value: `${elements.color.gray}` },
+      borderColor: { value: `${elements.color.gray.base}` },
+      background: { value: `${elements.color.gray.base}` },
       backgroundImage: { value: `url("${checkmarkSvg}")` },
     },
     checked: {
-      borderColor: { value: `${elements.color.primary}` },
-      background: { value: `${elements.color.primary}` },
+      borderColor: { value: `${elements.color.primary.base}` },
+      background: { value: `${elements.color.primary.base}` },
```

Comment:
There should be no `.base` color, we decided to use `.main`

Reply:
Fixed!